modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -159,26 +159,26 @@ $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_master-machi
 ----
 +
 By removing these files, you prevent the cluster from automatically generating control plane machines.
-endif::aws,azure,ash,gcp[]
+endif::azure,ash,gcp[]
 
-ifdef::aws,ash,azure,gcp[]
+ifdef::ash,azure,gcp[]
 . Remove the Kubernetes manifest files that define the control plane machine set:
 +
 [source,terminal]
 ----
 $ rm -f <installation_directory>/openshift/99_openshift-machine-api_master-control-plane-machine-set.yaml
 ----
-endif::aws,ash,azure,gcp[]
+endif::ash,azure,gcp[]
 
 ifdef::gcp[]
 ifndef::user-infra-vpc[]
 . Optional: If you do not want the cluster to provision compute machines, remove
 the Kubernetes manifest files that define the worker machines:
 endif::user-infra-vpc[]
 endif::gcp[]
-ifdef::aws,azure,ash,user-infra-vpc[]
+ifdef::azure,ash,user-infra-vpc[]
 . Remove the Kubernetes manifest files that define the worker machines:
-endif::aws,azure,ash,user-infra-vpc[]
+endif::azure,ash,user-infra-vpc[]
 ifdef::aws,azure,ash,gcp[]
```

Comment:
@dfitzmau For AWS, was it intentional to include the second CLI command block for step one and remove the lead-in wording? I am wondering if the `ifdef` on line 182 should remove `aws`.

 AWS is on the left and Azure is on the right:
![Screenshot from 2023-09-19 13-22-44](https://github.com/openshift/openshift-docs/assets/82119979/d63c3b0a-7dff-4e93-baeb-83a6f27ff8bc)

Reply:
I'll ask the SME to chime in here as some input on a thread lead to some confusion around due 4.14 updates, which are outside the scope of my PR, as my PR corrects existing content.